motorway/contrib/amazon_kinesis/ramps.py

```diff
@@ -120,7 +132,7 @@ def can_claim_shard(self, shard_id):
         for shard in shards:  # Update active worker cache
             if shard['worker_id'] not in active_workers:
                 heartbeats[shard['worker_id']] = shard['heartbeat']
-            if heartbeats[shard['worker_id']] == shard['heartbeat']:
+            if shard['heartbeat'] == shards_by_shard_id[shard['shard_id']]['heartbeat']:
```

Comment:
What does this change do?

Reply:
made it more readable by only using 1 dictionary with heartbeats.